subgraph.avalanche.yaml

```diff
@@ -0,0 +1,24 @@
+specVersion: 0.0.4
```

Comment:
I assume that we just all forgot to commit and push local updates of the lib + supported network.